tst/EnergyPlus/unit/SurfaceGeometry.unit.cc

```diff
@@ -5289,7 +5287,7 @@ TEST_F(EnergyPlusFixture, HeatBalanceIntRadExchange_SetupEnclosuresWithAirBounda
 
 }
 
-TEST_F(EnergyPlusFixture, HeatBalanceIntRadExchange_SetupEnclosuresWithAirBoundaries2)
+TEST_F(EnergyPlusFixture, DISABLED_HeatBalanceIntRadExchange_SetupEnclosuresWithAirBoundaries2)
```

Comment:
Is this unit test going to be fixed back up to work or will it be removed as a part of these changes?

Reply:
Not sure yet, may transform it to test something different.